apollo-router/src/spec/query/change.rs

```diff
@@ -499,14 +529,14 @@ mod tests {
 
         // id is nullable in 1, non nullable in 2
         let query = "query { me { id name } }";
-        assert_ne!(hash(schema1, query), hash(schema2, query));
+        assert!(hash(schema1, query) != hash(schema2, query));
```

Comment:
`assert_ne` is better because it shows two values, in case of `assert` we will just get true/false. So it is harder to debug, especially if it failed on CI.